src/components/CartItems/CartItems.test.js

```diff
@@ -12,7 +11,7 @@ describe('CartItems', () => {
       {
         id: '-MXGc7d2QzP8TTDWgBX_',
         name: 'test',
-        pictureUrl: 'https://test.img',
+        pictureUrl: 'https://ipsumimage.appspot.com/140x100.png',
```

Comment:
Okay, but did you correct regexp?
Or 'https://test.img' is still valid?

Reply:
yes i did)